src/components/Things.ts

```diff
@@ -10,7 +10,7 @@ import { Direction } from "./Constants";
 import { IWildPokemonSchema } from "./Maps";
 import { IDialog, IMenuSchema } from "./Menus";
 import { IStateSaveable } from "./Saves";
-
+import { Names } from "./things/names";
```

Comment:
Keep these imports together. No need for a blank line between them.

Also, keep a blank line underneath them so they're separate from the rest of the file.